### DISCUSSION TITLE
更改端口出现如下报错 此提示仅开发环境显示 端口有误: 查看README.md 开发调试

### DISCUSSION THREAD

**Discussion by serene-ding** [umijs/umi#12032] [Q&A]
![image](https://github.com/umijs/umi/assets/77049532/53d76457-06bd-4a57-ab4b-f87fa2c1f433)
我只是更改了端口 
![image](https://github.com/umijs/umi/assets/77049532/1071d295-347d-47c9-a3f1-622a6f95465c)
就出现图上报错

**Comment by fz6m** [comment]
这个报错和你的项目本身的设定有关，自己查看下项目的 readme 或者问一下相关同事吧。
